charts/tidb-operator/templates/advanced-statefulset-rbac.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: advanced-statefulset-controller
+  namespace: pingcap
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: advanced-statefulset-controller
+rules:
+- apiGroups:
+  - apps.pingcap.com
+  resources:
+  - '*'
+  verbs:
+  - '*'
+- apiGroups:
+  - 'apps'
+  resources:
+  - 'controllerrevisions'
+  verbs:
+  - '*'
+- apiGroups:
+  - ''
+  resources:
+  - 'pods'
+  - 'persistentvolumeclaims'
+  - 'persistentvolumes'
+  verbs:
+  - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: advanced-statefulset-controller
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: advanced-statefulset-controller
+subjects:
+- kind: ServiceAccount
+  name: advanced-statefulset-controller
+  namespace: pingcap
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: advanced-statefulset-controller
+  namespace: kube-system
+rules:
+- apiGroups:
+  - ''
+  resources:
+  - 'endpoints'
+  verbs:
+  - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: advanced-statefulset-controller
+  namespace: kube-system
```

Comment:
Why this ns is `kube-system`?

Reply:
testing only, this controller does not support specifying namespace (working on it)

Reply:
updated